addons/website_sale_slides/views/website_slides_templates.xml

```diff
@@ -68,6 +68,19 @@
 </template>
 
 <!-- COURSE -->
+<template name="Course Main" id="course_main" inherit_id="website_slides.course_main">
+    <xpath expr="//div[@id='home']" position="before">
+        <t t-set="channel_product" t-value="channel.product_id.sudo()"/>
+        <div t-if="not channel_product.is_published" class="alert alert-info" role="alert"
+            groups="website_slides.group_website_slides_officer">
+            This course cannot be bought because its linked product
+            <a t-attf-href="/web#id=#{channel_product.id}&amp;view_type=form&amp;model=#{channel_product._name}"
+                t-out="channel_product.name"/>
+            is not published.
```

Comment:
As `group_website_slides_officer` can read products (templates), we shouldn't need `sudo()` once we are in the `t-if` (and then there `t-set` may not be necessary anymore as well). Do we need it for parent or child templates?